tests/snapshots/assets__check_file@cetz-manual.typ-80.snap

```diff
@@ -62,9 +62,7 @@ Note that draw functions are imported inside the scope of the `canvas` block. Th
 #show raw.where(block: false): it => if it.text.starts-with("<") and it.text.ends-with(">") {
   set text(1.2em)
   doc-style.show-type(it.text.slice(1, -1))
-} else {
-  it
-}
+} else { it }
```

Comment:
This doesn't look good. I suggest we only try put them in a single line iff both branch of `if` has exactly one item